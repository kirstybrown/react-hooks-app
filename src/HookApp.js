import React from 'react';



const HookApp = () => {
  return (
      <div>
          <h1>Hello from HookApp</h1>
      </div>
  ) 
}

export default HookApp;
