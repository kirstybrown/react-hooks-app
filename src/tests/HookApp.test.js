import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import HookApp from '../HookApp';

describe('Tests for HookApp component', () => { 
    
    const wrapper = shallow( <HookApp />);

    test('should show Hook App correctly', () => { 
        expect( wrapper ).toMatchSnapshot();
     });

     test('should show header correctly', () => { 

        const title = 'Hello from HookApp';
        const h1Text = wrapper.find('h1').text().trim();
        expect( h1Text ).toBe( title );
      });
 });