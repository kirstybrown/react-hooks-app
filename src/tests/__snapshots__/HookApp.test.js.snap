// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for HookApp component should show Hook App correctly 1`] = `
<div>
  <h1>
    Hello from HookApp
  </h1>
</div>
`;
